src/python_framework_v02/troute/nhd_network_augment.py

```diff
@@ -848,25 +848,25 @@ def main():
         "exporting RouteLink file:",
         filename,
         "to",
-        os.path.join(root, "test", "input", "geo", "Channels"),
+        pathlib.Path(root, "test", "input", "geo", "Channels").resolve(),
     )
 
-    dir_path = os.path.join(root, "test", "input", "geo", "Channels", dirname)
-    if not os.path.isdir(dir_path):
-        os.mkdir(dir_path)
+    dir_path = pathlib.Path(root, "test", "input", "geo", "Channels", dirname).resolve()
+    # if not os.path.isdir(dir_path):
+    #     os.mkdir(dir_path)
```

Comment:
- [ ] @jhrehanoaa We'll want to find the way to use pathlib to retain this functionality.